packages/cli/src/cli.ts

```diff
@@ -14,9 +14,7 @@ import {
   BatchError,
 } from "./errors";
 
-// tricking static analysis is fun
-// @ts-ignore
-process["e" + "nv"].NODE_ENV = "production";
+process.env.NODE_ENV = "production";
```

Comment:
is this even needed now? I'm not sure what this was about

Reply:
It's because some babel plugins might rely on `process.env.NODE_ENV` and Preconstruct is basically always meant for building for "production"